src/tools/map_extractor/loadlib/DBFilesClientList.h

```diff
@@ -425,6 +425,12 @@ char const* DBFilesClientList[] =
     "DBFilesClient\\gtBattlePetTypeDamageMod.dbc",
     "DBFilesClient\\gtBarberShopCostBase.dbc",
     "DBFilesClient\\gtArmorMitigationByLvl.dbc",
+    "DBFilesClient\\gtNpcTotalHp.dbc",
```

Comment:
These changes should not be in this PR. See warning at the bottom of http://collab.kpsn.org/pages/viewpage.action?pageId=12714394